### ISSUE TITLE
Fix Travis CI

### ISSUE THREAD

**Issue by LeeTL1220** [broadinstitute/oncotator#346]
It cannot seem to setup the environment correctly in Travis.

**Comment by lbergelson** [comment]
Hmn, I see the same error when I try to install locally on your new branch.  


(this is from travis but I'm seeing the same thing)
```
creating /home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/distribute-0.6.15-py2.7.egg
Extracting distribute-0.6.15-py2.7.egg to /home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages
Removing distribute 0.7.3 from easy-install.pth file
Adding distribute 0.6.15 to easy-install.pth file
Installing easy_install script to /home/travis/virtualenv/python2.7.9/bin
Installing easy_install-2.7 script to /home/travis/virtualenv/python2.7.9/bin
Installed /home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/distribute-0.6.15-py2.7.egg
Traceback (most recent call last):
  File "setup.py", line 142, in <module>
    'Topic :: Scientific/Engineering :: Medical Science Apps.',
  File "/opt/python/2.7.9/lib/python2.7/distutils/core.py", line 151, in setup
    dist.run_commands()
  File "/opt/python/2.7.9/lib/python2.7/distutils/dist.py", line 953, in run_commands
    self.run_command(cmd)
  File "/opt/python/2.7.9/lib/python2.7/distutils/dist.py", line 972, in run_command
    cmd_obj.run()
  File "build/bdist.linux-x86_64/egg/setuptools/command/install.py", line 73, in run
  File "build/bdist.linux-x86_64/egg/setuptools/command/install.py", line 101, in do_egg_install
  File "build/bdist.linux-x86_64/egg/setuptools/command/easy_install.py", line 344, in run
    
  File "build/bdist.linux-x86_64/egg/setuptools/command/easy_install.py", line 564, in easy_install
    
  File "build/bdist.linux-x86_64/egg/setuptools/command/easy_install.py", line 616, in install_item
    
  File "build/bdist.linux-x86_64/egg/setuptools/command/easy_install.py", line 663, in process_distribution
    
TypeError: __init__() takes exactly 2 arguments (4 given)
The command "python setup.py install" failed and exited with 1 during .
```

**Comment by LeeTL1220** [comment]
Closed by #344
